client/src/components/SelfReviewForm/SelfReviewForm.js

```diff
@@ -0,0 +1,320 @@
+  const selfReviewErrorMessage = useSelector(selfReviewCreateErrorSelector)
+  const selfReviewUpdateStatus = useSelector(selfReviewUpdateSelector)
+  const selfReviewUpdateError = useSelector(selfReviewUpdateErrorSelector)
+  const dispatch = useDispatch()
+  useEffect(() => {
+    dispatch(loadAllProjects())
+    dispatch(loadAllEmployeeData())
+    dispatch(loadManagers())
+  }, [dispatch])
+  useEffect(() => {
+    if (selfReviewSuccessMessage) {
+      addToast('Self Review successfully added', {
+        appearance: 'success',
+        autoDismiss: true
+      })
+      dispatch(setSelfReviewSuccess(''))
+      clickHandler()
+    }
+  }, [selfReviewSuccessMessage, addToast, dispatch, clickHandler])
+
+  useEffect(() => {
+    if (selfReviewErrorMessage) {
+      addToast('Error while saving form', {
+        appearance: 'error',
+        autoDismiss: true
+      })
+      dispatch(setSelfReviewError(''))
+    }
+  }, [selfReviewErrorMessage, addToast, dispatch, clickHandler])
+
+  useEffect(() => {
+    if (selfReviewUpdateStatus) {
+      addToast('Self Review successfully updated', {
+        appearance: 'success',
+        autoDismiss: true
+      })
+      clickHandler()
+      dispatch(setUpdateReviewStatus(''))
+    }
+  }, [selfReviewUpdateStatus, addToast, dispatch, clickHandler])
+
+  useEffect(() => {
+    if (selfReviewUpdateError) {
+      addToast('Error while saving form', {
+        appearance: 'error',
+        autoDismiss: true
+      })
+      dispatch(setUpdateReviewError(''))
+    }
+  }, [selfReviewUpdateError, addToast, dispatch])
+  const submitReview = values => {
+    if (selfReviewInfo) {
+      dispatch(updateSelfReview(selfReviewInfo._id, values))
+    } else {
+      dispatch(createSelfReview(values))
+    }
+  }
+  return (
+    <Grid>
+      <Formik
+        initialValues={initialValues(selfReviewInfo)}
+        onSubmit={(values, { setSubmitting }) => {
+          submitReview(values)
+          setSubmitting(false)
+        }}
+        validationSchema={Yup.object(validationSchema)}
+      >
+        {({ isSubmitting, values, setFieldValue, handleChange }) => (
+          <Card>
+            <Form>
+              <CardHeader color="primary">
+                <h4 className={classes.cardTitleWhite}>CREATE SELF REVIEW FORM</h4>
+              </CardHeader>
+              <CardBody>
+                <Grid className={classes.container} container>
+                  <Grid xs={6} sm={6} md={3} className={classes.grid} item>
+                    Employee *
+                  </Grid>
+                  <Grid xs={6} sm={6} md={3} item>
+                    <SelectMenu
+                      name="employee"
+                      onChange={handleChange}
+                      disabledName="Select Employee"
+                      value={values.employee}
+                    >
+                      {employeeData
+                        ? employeeData.map((prop, key) => {
```

Comment:
if you want render only true condition.  you can use && (AND) operator. also you no need to return **null**.
EX. {employeeData
        && employeeData.map((prop, key) => {
          return prop.status !== 'Inactive' && (
            <MenuItem
              className={classes.hoverEffect}
              value={prop._id}
              key={key}
            >
              {prop.firstname} {prop.lastname}
            </MenuItem>
          )
        })
        }

Reply:
Done